projects/igniteui-angular/src/lib/progressbar/progressbar.component.spec.ts

```diff
@@ -0,0 +1,209 @@
+import { Component } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { BaseProgressDirective } from './progressbar.component';
+
+@Component({
+    template: '',
+})
```

Comment:
```suggestion
@Directive()
```

Reply:
I am unable to resolve this, if I switch to directive I get an error(Error: It looks like 'TestProgressDirective' has not been compiled) i am importing the directive in the  configureTestingModule imports but i still get the error, i am unable to resolve this one.